utils/run_test.sh

```diff
@@ -148,17 +148,11 @@ if [ -d "/mnt/daos" ]; then
 
         rm -f "${AIO_DEV}"
         dd if=/dev/zero of="${AIO_DEV}" bs=1G count=20
```

Comment:
Does this still test MD_ON_SSD?   It looks identical to the run on 147 except with bigger AIO file.  Previously, it was using PMDK for one and NVMD_ON_SSD for the other.